swap_meet/item.py

```diff
@@ -1,2 +1,28 @@
+
+
+
+
 class Item:
-    pass
+    
+    def __init__(self, condition = 0, category = ""):
+        if category:
+            self.category = category
+        else:
+            self.category = ''
+        if condition :
+            self.condition = condition
+        else:
+            self.inventory = 0
```

Comment:
The default parameters take care of falsy values so these conditionals aren't needed either ( along with adding an inventory attribute). This means that if None is passed in as the `condition` value then 0 will automatically be assigned to `condition`. 

We can refactor like so:

```suggestion
        self.category = category
        self.condition = condition if condition is not None else 0
```